secrets/secrets.go

```diff
@@ -110,8 +110,6 @@ func parseSecretsLookupURN(ctx context.Context, logger *slog.Logger, urn string)
 		return newSecretManager(ctx, logger, u, newGCPSecretsManager)
 	case "az":
 		return newSecretManager(ctx, logger, u, newAzSecretsManager)
-	case "redis":
```

Comment:
Can we also get rid of the whole URN stuff here too? We don't need to stuff everything into a string in cloudv2 repo - we only do that in connect because it's a commandline arg.

Ideally these are just the implementations of aws, gcp and az secret managers that satisfy some interface in the connect repo and can be plugged in there.